src/researchhub_document/related_models/researchhub_unified_document_model.py

```diff
@@ -215,10 +215,38 @@ def created_by(self):
             return None
 
     def get_review_details(self):
-        details = {"avg": 0, "count": 0}
-        reviews = self.reviews
+        """Return average score & count of *active* reviews.
+
+        A review is considered active iff:
+        1. The review itself has not been soft-deleted (`is_removed=False`).
+        2. The underlying comment (`RhCommentModel`) it references has not been
+           soft-deleted.
+
+        Because `Review.item` is a `GenericForeignKey`, we cannot traverse the
+        relationship using the ORM double-underscore (`item__…`).  Instead we
+        filter by `content_type` and `object_id`.
+        """
+
+        from django.contrib.contenttypes.models import ContentType
+
+        from researchhub_comment.related_models.rh_comment_model import RhCommentModel
```

Comment:
Not: Importing these common imports at the top should be fine and will improve readability.